src/SFA.DAS.ContentApi/Application/Queries/GetContentQuery/GetContentQueryHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.ContentApi.Data;
+
+namespace SFA.DAS.ContentApi.Application.Queries.GetContentQuery
+{
+    public class GetContentQueryHandler : IRequestHandler<GetContentQuery, string>
+    {
+        private readonly Lazy<ContentApiDbContext> _db;
+
+        public GetContentQueryHandler(Lazy<ContentApiDbContext> db)
+        {
+            _db = db;
+        }
+
+        public async Task<string> Handle(GetContentQuery request, CancellationToken cancellationToken)
```

Comment:
Rather than return a string it would be better to return an object e.g. GetContentResponse with a Data or Content property.  Then we have a structure that can be extended going forward as requirements change without breaking the interface.

Reply:
Discuss with Corey. That was the original implementation.

Reply:
Actually.. Agree I'll make the change.